### ISSUE TITLE
Timeline in 'Programs' page is breaking

### ISSUE THREAD

**Issue by iamkryptonite** [anitab-org/anitab-org.github.io#246]
### Describe the bug
The timeline in 'Programs' page is not working. The marker does not show the current date at the correct place in the timeline. Instead the marker keeps looping in one place. For example if todays date is 3rd March, instead of showing it in the appropriate place in the timeline it will be shown in '3rd January' .

### To Reproduce
1. Run the local server on your device 
2. Click on 'Programs'

### Expected behavior
The marker in timeline is suppose to show the current date and month in the correct place and should keep progressing everyday smoothly.

### Screenshots
![AnitaB_org_open_source](https://user-images.githubusercontent.com/43510609/109451551-ae5b9f80-7a73-11eb-93cd-582627461319.png)

### Desktop (please complete the following information):
 - OS: Windows
 - Browser: Firefox

**Comment by prathameshbhalekar** [comment]
Can I work on it if approved?

**Comment by iamkryptonite** [comment]
> Can I work on it if approved?

Sure, just wait for it to be approved. Meanwhile you can go through pr #150 because this issue is related to it.

**Comment by nandini45** [comment]
@iamkryptonite  yeah i will approve this issue.
@prathameshbhalekar  do u want to work on it?

**Comment by prathameshbhalekar** [comment]
> @iamkryptonite yeah i will approve this issue.
> @prathameshbhalekar do u want to work on it?

Yeah but my exams are going on right now so I'll need a week to work on it.

**Comment by nandini45** [comment]
@prathameshbhalekar  i will assign you it but it will be still available.
 if anyone comes and ask for the work i will assign that person
if after a week still no one works on it 
you can work it

**Comment by nlok5923** [comment]
@nandini45 can i take this issue.

**Comment by nandini45** [comment]
@nlok5923  ok

**Comment by iamkryptonite** [comment]
@nlok5923 I have updated the issue with what the 'timeline' should look like. What you need to do is adjust the positions according to the current date and month. If you need any help please ask.

**Comment by nlok5923** [comment]
thanks @iamkryptonite working on it.

**Comment by shruti1421** [comment]
Can I try this issue?

**Comment by Rahulm2310** [comment]
@keshakaneria Can you please remove the `Available` label here and other issues wherever a contributor is assigned so that it isn't misleading for contributors.
